# HG changeset patch
# User Rik <rik@octave.org>
# Date 1479164318 28800
#      Mon Nov 14 14:58:38 2016 -0800
# Branch stable
# Node ID 6033e2e4d5ea8a2420c531c19d997c7baddd9c79
# Parent  1c67938a15f10305ce080f8100507f1de4259c78
doc: Clarify inputs and outputs for rgb2hsv.

* rgb2hsv.m: Document inputs of hsv_img and hsv_map to clarify what function
operates on.

diff --git a/scripts/image/rgb2hsv.m b/scripts/image/rgb2hsv.m
--- a/scripts/image/rgb2hsv.m
+++ b/scripts/image/rgb2hsv.m
@@ -12,18 +12,18 @@
 ## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
-## @deftypefn  {} {@var{hsv_map} =} rgb2hsv (@var{rgb})
-## @deftypefnx {} {@var{hsv_map} =} rgb2hsv (@var{rgb})
+## @deftypefn  {} {@var{hsv_map} =} rgb2hsv (@var{rgb_map})
+## @deftypefnx {} {@var{hsv_img} =} rgb2hsv (@var{rgb_img})
 ## Transform a colormap or image from RGB to HSV color space.
 ##
 ## A color in the RGB space consists of red, green, and blue intensities.
 ##
 ## A color in HSV space is represented by hue, saturation and value
 ## (brightness) levels in a cylindrical coordinate system.  Hue is the
 ## azimuth and describes the dominant color.  Saturation is the radial
 ## distance and gives the amount of hue mixed into the color.  Value is
